tsconfig.json

```diff
@@ -8,9 +8,10 @@
 		"resolveJsonModule": true,
 		"skipLibCheck": true,
 		"sourceMap": true,
-		"strict": true,
-		"types": ["vitest/globals", "@testing-library/jest-dom"]
-	}
+		"strict": true
+	},
+	"include": ["**/*.pw.ts"]
```

Comment:
Do we need this? Since we no longer use .pw.ts files

Reply:
Changed to .test. The line solves an issue where tests show typescript import errors after running playwright tests, unless this line is here. I think during the build step for playwright, the types gets messed up, and this line helps vscode language server in recovering.

Reply:
I ran into that issue as well, thanks for the fix!